### PR TITLE
Added Chartboost.Framework to Github project

### PR DESCRIPTION
**Pull request by arnaumartifont**
Added Chartboost framework to the github project, removed from .gitignore

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ The ChartboostExample project includes two targets:
 
 - `ChartboostExample`: Manual dependency management. 
 
- It requires the `Chartboost.framework` to be dropped into the root directory (where this readme is).
+It requires the `Chartboost.framework`, the lastest version is already in the root directory (where this readme is).
```
Comment (daniel-barros) [inline README.md:24]: `... readme is) and linked in the Xcode project.` 
To make clear that nothing needs to be done?
Also, "the lastest version" will get outdated very quickly. Maybe something like:
"The `Chartboost.framework` is already in the root directory (where this readme is) and linked to the Xcode project."
?

Comment (arnaumartifont) [reply README.md:24]: yes, better that way, more clear 👍🙌

File: README.md [hunk 2]
```diff
@@ -21,7 +21,7 @@ The ChartboostExample project includes two targets:
 
 - `ChartboostExample`: Manual dependency management. 
 
- It requires the `Chartboost.framework` to be dropped into the root directory (where this readme is).
+It requires the `Chartboost.framework`, is already in the root directory (where this readme is) and linked to the Xcode project.
```
Comment (daniel-barros) [inline README.md:24]: A small grammar correction:
"It requires the Chartboost.frameworks, **which is** already in..."

**Review by daniel-barros — APPROVED** [review]
Nice 👏

**Pull request merged**